handler_error.go

```diff
@@ -87,6 +88,12 @@ func (e *ErrorHandler) HandleError(w http.ResponseWriter, r *http.Request, errMs
 
 		t := time.Now()
 
+		if ctxGetDefaultVersion(r) {
+			if vinfo := ctxGetVersionInfo(r); vinfo != nil {
+				w.Header().Set("X-Tyk-Default-Version", vinfo.Name)
```

Comment:
I assume you forgot replace constant here

Reply:
Good spot